store/src/java/com/zimbra/cs/service/util/JWTUtil.java

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
Please add detailed comments for each of the public methods.
Also add logging statements, which might be useful while debugging.

Add unit test for the public utility methods

Reply:
done.